src/locales/en/translation.json

```diff
@@ -240,9 +240,13 @@
       "description": "Please try another search query <OptionalBreak>or <HomeLink>explore Oasis</HomeLink></OptionalBreak>"
     },
     "otherResults": {
-      "clickToShowThem": "We found <strong>{{ countLabel }}</strong> on the <strong>{{ networkName }}</strong>. Click to open these results.",
-      "clickToShowMore": "We also found <strong>{{ countLabel }}</strong> on the <strong>{{ networkName }}</strong>. Click to open these results.",
-      "clickToHide": "The {{ networkName }} results are shown below. Click to hide those results."
+      "otherNetworks": "other <strong>Networks</strong>",
+      "otherParatimes": "other <strong>Paratimes</strong>",
+      "otherNetworksAndParatimes": "other <strong>Networks and Paratimes</strong>",
+      "otherParatimesOnNetwork": "Other Paratimes on {{ network }}",
+      "clickToShowThem": "We found <strong>{{ countLabel }}</strong> on {{ location }}. Click to open these results.",
+      "clickToShowMore": "We also found <strong>{{ countLabel }}</strong> on {{ location }}. Click to open these results.",
```

Comment:
This might be simplifiable https://www.i18next.com/translation-function/formatting#list
`We found <strong>{{ countLabel }}</strong> on other <strong>{{scopes, list}}</strong>. Click to open these results.`

Reply:
Simplified as suggested.